invenio_app_rdm/records_ui/views/records.py

```diff
@@ -71,13 +71,27 @@ def open(self):
 #
 # Views
 #
+def load_custom_fields(conf_ui, conf_backend):
```

Comment:
shelvable: we had a similar loader somewhere else for the react part? I wonder if it could be reused/made generic.